src/batch/src/executor/insert.rs

```diff
@@ -231,8 +231,8 @@ impl BoxedExecutorBuilder for InsertExecutor {
                 .map(|IndexAndExpr { index: i, expr: e }| {
                     Ok((
                         i as usize,
-                        build_from_prost(&e.ok_or_else(|| anyhow!("expression is None"))?)
-                            .map_err(|e| anyhow!("failed to build expression: {}", e))?,
+                        build_from_prost(&e.context("expression is None")?)
```

Comment:
Besides `Result`, `context` is also applicable to `Option`s.

If you only have the error, use `anyhow::anyhow!("message").context(source)`.